tests/host/lldb/launch_guest.py

```diff
@@ -25,23 +25,39 @@ class _LLDBController(Controller):
         def __init__(self, pc: PwndbgController):
             self.pc = pc
 
-        async def launch(self, binary: Path) -> None:
+        async def launch(self, binary: Path, args: List[str] = []) -> None:
             await self.pc.execute(f"target create {binary}")
-            await self.pc.execute("process launch -s")
+            await self.pc.execute("process launch -s -- " + " ".join(args))
```

Comment:
```suggestion
            await self.pc.execute("process launch -s -- " + " ".join(shlex.quote(arg) for arg in args))
```
probably this way we should handle it?

Reply:
Maybe. Originally I chose not to do it because I felt leaving that up to the tests themselves was good enough, and, indeed, over the course of porting the tests it never came up. But if you feel it's better to add quotations automatically, I'm okay with it, too, and it might be helpful for people adding new tests in the future.